src/components/FilmstripViewer.jsx

```diff
@@ -0,0 +1,124 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import SVGImage from './SVGImage';
+import { connect } from 'react-redux';
+import { getAllLocations } from '../lib/get-subject-location';
+import { changeFrame } from '../ducks/subject-viewer';
+
+class FilmstripViewer extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.renderFrames = this.renderFrames.bind(this);
+    this.scroll = this.scroll.bind(this);
+  }
+
+  changeFrame(i) {
+    this.props.dispatch(changeFrame(i))
+  }
+
+  scroll(scrollDown = true) {
+    let i = 0;
+
+    const scroll = () => {
+      scrollDown ? this.viewport.scrollTop += 20 : this.viewport.scrollTop -= 20
+      i += 1;
+
+      if (i < 10) { setTimeout(scroll, 20); }
+    }
+    scroll();
+  }
+
+  renderFrames() {
+    const SVG_WIDTH = 40;
+    const SVG_HEIGHT = 60;
+    let scale = 0.08;
+    const viewBox = `-${(SVG_WIDTH / scale) / 2} -${(SVG_HEIGHT / scale) / 2} ${SVG_WIDTH / scale} ${SVG_HEIGHT / scale}`;
+    if (!this.props.currentSubject) { return }
+    const images = getAllLocations(this.props.currentSubject);
+
+    const render = images.map((image, i) => {
+      const activeBorder = i === this.props.frame ? "related-images__frame--active" : ""
+
+      return (
+        <div key={i} className={`related-images__frame ${activeBorder}`} ref={(el) => {this.strip = el; }}>
+          <button onClick={this.changeFrame.bind(this, i)}>
+            <svg
+              style={{ width: `${SVG_WIDTH}px`, height: `${SVG_HEIGHT}px` }}
+              ref={(c) => { this.svg = c; }}
+              viewBox={viewBox}
+            >
+              <g>
+                <SVGImage
+                  ref={(c) => { this.svgImage = c; }}
+                  src={image.src}
+                />
+              </g>
+            </svg>
+            <div>
+              <span> {i + 1} / {images.length}</span>
+            </div>
+          </button>
+        </div>
+      )
+    })
+    return (
+      <div className="related-images__frames">
+        {render}
+      </div>
+    )
+  }
+
+  render() {
+    const renderButtons = this.props.currentSubject && this.props.currentSubject.locations.length > 5 ? true : false;
```

Comment:
The `true : false` is redundant here since that's the value of the conditional on the left of the question mark.

I think you could just assign `renderButtons = this.props.currentSubject && this.props.currentSubject.locations.length > 5`